frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -316,6 +327,10 @@ export default {
 
   // reload data every time user navigates to Activity view
   async mounted() {
+    const localDisplaySize = localStorage.getItem('activityDisplaySize')
+    this.displaySize = ['paper', 'widescreen'].includes(localDisplaySize)
+      ? localDisplaySize
+      : 'paper'
```

Comment:
maybe you can move the default into a computed and use it from there.
Then its quickly changed in all places